.github/CODEOWNERS

```diff
@@ -1,2 +1,3 @@
 # This should match the owning team set up in https://github.com/orgs/opensearch-project/teams
-*   @pjfitzgibbons @ps48 @kavithacm @derek-ho @joshuali925 @dai-chen @YANG-DB @rupal-bq @mengweieric @vamsi-amazon @swiddis @penghuo @seankao-az @anirudha @paulstn @sumukhswamy
+
+- @pjfitzgibbons @ps48 @kavithacm @derek-ho @joshuali925 @dai-chen @YANG-DB @rupal-bq @mengweieric @vamsi-amazon @swiddis @penghuo @seankao-az @anirudha @paulstn @sumukhswamy @sejli @TackAdam @RyanL1997
```

Comment:
Please fix the codeowners file for @pjfitzgibbons
Either need to retain the maintainer status or move them to emeritus section.

Reply:
Yes ^^

Reply:
updated, thanks!